invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/AdminForm.js

```diff
@@ -15,13 +15,17 @@ import mapValues from "lodash/mapValues";
 export class AdminForm extends Component {
   constructor(props) {
     super(props);
-    const { resource, formFields } = props;
+    const { resource, resourceSchema: resourceSchema } = props;
```

Comment:
gh actions is complaining about "Destructuring assignment resourceSchema unnecessarily renamed". Can you check that please?

Reply:
I see this is duplicated, I am closing it :)